src/Entity/IdeasWorkshop/Answer.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace AppBundle\Entity\IdeasWorkshop;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\Mapping as ORM;
+use Algolia\AlgoliaSearchBundle\Mapping\Annotation as Algolia;
+
+/**
+ * @ORM\Table(name="ideas_workshop_answer")
+ * @ORM\Entity
+ *
+ * @Algolia\Index(autoIndex=false)
+ */
+class Answer
+{
+    /**
+     * @var int
+     *
+     * @ORM\Column(type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="text")
+     */
+    private $content;
+
+    /**
+     * @ORM\ManyToOne(targetEntity="Question")
+     */
+    private $question;
+
+    /**
+     * @ORM\OneToMany(targetEntity="Thread", mappedBy="answer")
+     */
+    private $threads;
+
+    /**
+     * @ORM\ManyToOne(targetEntity="Idea", inversedBy="answers")
+     */
+    private $idea;
+
+    public function __construct(
+        string $content,
+        Question $question
```

Comment:
`Idea` object is missing

Reply:
Why do I need to add Idea here if the owning side is on Idea ? :thinking: